[2024-09-03T22:57:50.891+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: SitesWebScraping.WebScrapint manual__2024-09-03T22:57:49.024813+00:00 [queued]>
[2024-09-03T22:57:50.904+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: SitesWebScraping.WebScrapint manual__2024-09-03T22:57:49.024813+00:00 [queued]>
[2024-09-03T22:57:50.905+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-09-03T22:57:50.920+0000] {taskinstance.py:2192} INFO - Executing <Task(WebScraping_K): WebScrapint> on 2024-09-03 22:57:49.024813+00:00
[2024-09-03T22:57:50.926+0000] {standard_task_runner.py:60} INFO - Started process 145 to run task
[2024-09-03T22:57:50.929+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'SitesWebScraping', 'WebScrapint', 'manual__2024-09-03T22:57:49.024813+00:00', '--job-id', '361', '--raw', '--subdir', 'DAGS_FOLDER/DagWebScraping.py', '--cfg-path', '/tmp/tmp3dcjjfqd']
[2024-09-03T22:57:50.934+0000] {standard_task_runner.py:88} INFO - Job 361: Subtask WebScrapint
[2024-09-03T22:57:50.982+0000] {task_command.py:423} INFO - Running <TaskInstance: SitesWebScraping.WebScrapint manual__2024-09-03T22:57:49.024813+00:00 [running]> on host 6d24e5a510c9
[2024-09-03T22:57:51.105+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='SitesWebScraping' AIRFLOW_CTX_TASK_ID='WebScrapint' AIRFLOW_CTX_EXECUTION_DATE='2024-09-03T22:57:49.024813+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-03T22:57:49.024813+00:00'
[2024-09-03T22:57:51.112+0000] {taskinstance.py:2699} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 1182, in execute
    raise NotImplementedError()
NotImplementedError
[2024-09-03T22:57:51.136+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=SitesWebScraping, task_id=WebScrapint, execution_date=20240903T225749, start_date=20240903T225750, end_date=20240903T225751
[2024-09-03T22:57:51.183+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 361 for task WebScrapint (; 145)
[2024-09-03T22:57:51.237+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-09-03T22:57:51.276+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
